mlflow/tracking/fluent.py

```diff
@@ -514,7 +514,7 @@ def get_run(run_id: str) -> Run:
     return MlflowClient().get_run(run_id)
 
 
-def log_param(key: str, value: Any) -> None:
+def log_param(key: str, value: Any) -> Any:
```

Comment:
Can we update the docstring and example?

Reply:
Sure.